.github/workflows/Release-Please.yml

```diff
@@ -0,0 +1,67 @@
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: Run Release Please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    environment: deployment
+    strategy:
+      matrix:
+        node-version: [ 18.x ]
+    steps:
+      - uses: google-github-actions/release-please-action@v2
+        id: release
+        with:
+          command: manifest
+          token: ${{secrets.GITHUB_TOKEN}}
+          default-branch: main
+          monorepo-tags: true
+
+
+  Setup-Environment:
+    runs-on: ubuntu-latest
+    if: ${{ steps.release.outputs.releases_created }}
+    strategy:
+      matrix:
+        node-version: [ 18.x ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
Could be a good idea to pass the `ref` to access the tag that was created by release please.  
```
with:
  ref: ${{ needs.release-please.outputs.release_tag_name }}
// or 
   ref: ${{ steps.release.outputs.release_tag_name }}
```